ts/Extensions/Annotations/Popup/PopupAnnotations.ts

```diff
@@ -154,14 +154,25 @@ function addToolbar(
     }
 
     // create label
-    createElement('span', void 0, void 0, popupDiv).appendChild(
-        doc.createTextNode(pick(
-            // Advanced annotations:
-            lang[options.langKey as any] || options.langKey,
-            // Basic shapes:
-            options.shapes && options.shapes[0].type,
-            ''
-        ))
+    let label = createElement(
+        'p',
+        {
+            className: 'highcharts-annotation-label'
+        },
+        void 0,
+        popupDiv
+    );
+    label.setAttribute('aria-label', 'Annotation type');
```

Comment:
The `createElement` function takes attributes as the second argument. It should work with the `aria-label` attribute too...

Reply:
Yes, I've tried with the second argument, and something was broken. This is one point, which I have to test out, to see, how exactly can I add the `aria-label` to the correct element. Thanks!